shared/templates/audit_rules_privileged_commands/oval.template

```diff
@@ -23,7 +23,7 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="object_{{{ ID }}}_augenrules" version="1">
     <ind:filepath operation="pattern match">^/etc/audit/rules\.d/.*\.rules$</ind:filepath>
-{{% if product in ["sle12", "sle15"] %}}
+{{% if product in ["rhel8", "sle12", "sle15"] %}}
     <ind:pattern operation="pattern match">^[\s]*-a[\s]+always,exit[\s]+-F[\s]+path={{{ PATH }}}(?:[\s]+-F[\s]+perm=x)?[\s]+-F[\s]+auid>={{{ auid }}}[\s]+-F[\s]+auid!=(?:4294967295|unset)[\s]+(?:-k[\s]+|-F[\s]+key=)[\S]+[\s]*$</ind:pattern>
```

Comment:
Maybe the OVAL check should be changed from `(?:[\s]+-F[\s]+perm=x)?` to `(?:[\s]+-F[\s]+perm=x)` ?

Reply:
Yes, I'll just double check the STIG from SLE to see how this is handled there.

Reply:
It should be ok to have it mandatory and not optional.